iphone/Classes/TiUIScrollableView.m

```diff
@@ -169,6 +173,12 @@ -(void)layoutSubviews
 [wrapperView setViewName: TI_STRING(@"scrollable.wrapper.view%lu", (unsigned long)[[self subviews] count])]; \
 [wrapperView addSubview:view]; \
 
+
+-(void)removeSubview:(nonnull UIView *)view
+{
+    [[view superview] removeFromSuperview];
```

Comment:
the superview in this regard, is NOT scrollableview. It's actually a TiLayoutView, that is an extra layer BETWEEN the scrollableview and said subview, when auto layout is enabled. :)